app/views/administrate/application/new.html.erb

```diff
@@ -18,7 +18,7 @@ to do the heavy lifting.
 <% content_for(:title) do %>
   <%= t(
     "administrate.actions.new_resource",
-    name: display_resource_name(page.resource_name).titleize
+    name: page.resource_name.titleize
```

Comment:
Is there a corresponding test for this, too? If not, would you be able to contribute one?

Reply:
Will do

Reply:
Looks like it didn't work with i18n translated model name